Client/Shared/UserProfile/ProfileAnswers.razor.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Quanda.Client.Shared.UserProfile
+{
+    public partial class ProfileAnswers
+    {
+        protected override Task OnInitializedAsync()
+        {
+            throw new NotImplementedException("Waiting for UI template");
```

Comment:
#102 Has answer. Do you want to create separate task and implement answers on profile later?

Reply:
Yes in next small pull request